tests/testlibrary.c

```diff
@@ -285,9 +290,36 @@ test_remote (void)
   g_assert_cmpstr (flatpak_remote_get_collection_id (remote), ==, NULL);
   flatpak_remote_set_collection_id (remote, "org.example.CollectionID");
   g_assert_cmpstr (flatpak_remote_get_collection_id (remote), ==, "org.example.CollectionID");
+
+  /* Flatpak doesn't provide access to gpg-verify-summary, so use ostree */
+  res = flatpak_installation_modify_remote (inst, remote, NULL, &error);
+  g_assert_true (res);
+  g_assert_no_error (error);
```

Comment:
Nitpick: It’s generally better to put the `g_assert_no_error()` before the `g_assert_true()` so that you get more useful log messages out of the test in the case of failure.